src/core/support/string.c

```diff
@@ -174,6 +188,27 @@ char *gpr_strjoin(const char **strs, size_t nstrs, size_t *final_length) {
   return out;
 }
 
+static void do_nothing(void *ignored) {}
+gpr_slice_buffer *gpr_strsplit(const char *str, const char *sep) {
```

Comment:
Sigh... the latest commit includes an implementation of the latter option. However, it's not practical (ie, it's buggy) because of its use of strstr(), which relies on both its arguments being nil-terminated, which is not the case for slices. I'd rather not re-implement strstr() for slices, but I don't see a way around it (it's also late, which doesn't help) other than creating temporary c strings (incurring in allocs). Ideas?